app_dart/lib/src/service/config.dart

```diff
@@ -31,12 +34,18 @@ const String kDefaultBranchName = 'master';
 
 interface class Config {
   /// Creates and returns a [Config] instance.
-  Config(this._cache, this._secrets, {required DynamicConfig dynamicConfig})
-    : _dynamicConfig = dynamicConfig;
+  Config(
+    this._cache, //
+    this._secrets, {
+    required DynamicConfig dynamicConfig,
+  }) : _dynamicConfig = dynamicConfig;
 
   /// Access dynamically configured flags.
   DynamicConfig get flags => _dynamicConfig;
 
+  /// See [CiYaml.onlyUseTipOfTreeTargetsExistenceToFilterTargets].
+  static const bool kOnlyUseTipOfTreeTargetsExistenceToFilterTargets = false;
```

Comment:
Rather than exposing the default and maybe hiding places that do not pass this flag (error prone); make people refer to `config.onlyUseTipOfTreeTargetsExistenceToFilterTargets`.